gallery/__init__.py

```diff
@@ -476,6 +476,13 @@ def hide_file(file_id: int, auth_dict: Optional[Dict[str, Any]] = None):
         return "Permission denied", 403
 
     file_model.hidden = True
+
+    # Remove hidden file from directory thumbnails
+    for d in Directory.query.filter(Directory.thumbnail_uuid == file_model.thumbnail_uuid).all():
+        d.thumbnail_uuid = DEFAULT_THUMBNAIL_NAME
+
+    refresh_thumbnail()
```

Comment:
@RamZallan do thumbnails get refreshed periodically or does it need to be explicitly done this way every time?

Reply:
AFAIK we never refresh the thumbnail, though there is a CLI command exposed [here](https://github.com/ComputerScienceHouse/gallery/blob/f63e7e12ef8876d994fb6e7c756271f1ab749858/gallery/__init__.py#L311).

Reply:
Then I guess this does need to be run every time, unless we want to set up some timed job to do this.